src/main.ts

```diff
@@ -306,6 +311,9 @@ export async function activate(
     if (!useOmnisharpServer) {
         tryGetCSharpDevKitExtensionExports(csharpLogObserver);
 
+        const languageServerExport = new RoslynLanguageServerExport(
+            <Promise<RoslynLanguageServer>>roslynLanguageServerPromise
```

Comment:
```suggestion
            roslynLanguageServerPromise!
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator implies that works like in C#, right?